pom.xml

```
@@ -68,7 +68,7 @@
 
   <properties>
     <gitRepositoryName>sonar-scanner-ant</gitRepositoryName>
-    <artifactsToPublish>${project.groupId}:sonarqube-ant-task:jar</artifactsToPublish>
+    <artifactsToPublish>${project.groupId}:sonarqube-ant-task:jar,${project.groupId}:${project.artifactId}:json:cyclonedx</artifactsToPublish>
```

The artifact ID is different. Should we align them? Don't know what the best practice is here.

Like: 

```suggestion
    <artifactsToPublish>${project.groupId}:sonarqube-ant-task:jar,${project.groupId}:sonarqube-ant-task:json:cyclonedx</artifactsToPublish>
```

---

this is an error and would have generated a wrongly named bom, not critical but messy, thanks